jwtclient/cmd/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/dcos/dcos-go/jwtclient"
+)
+
+var (
+	flagURL       = flag.String("url", "", "URL to query")
+	flagIAMConfig = flag.String("iam-config", "", "Path to IAM config")
```

Comment:
F-ing new review process in github, my second comment never made it into the stream since I didn't "submit"